source/js/jquery-sortable.js

```diff
@@ -401,10 +402,11 @@
       ) >= this.options.distance)
     },
     getPointer: function(e) {
-      var o = e.originalEvent || e.originalEvent.touches && e.originalEvent.touches[0]
```

Comment:
This line doesn't do what the author probably meant for it to do. Note that `e.originalEvent` is always truthy, so this line is equivalent to `var o = e.originalEvent`.